tests/basic.rs

```diff
@@ -0,0 +1,26 @@
+use mkl_sys::{
+    _MKL_DSS_HANDLE_t, dss_create_, dss_delete_, MKL_DSS_DEFAULTS, MKL_DSS_ZERO_BASED_INDEXING,
+};
+use std::ptr::null_mut;
+
+#[test]
+fn does_run() {
```

Comment:
Might adding a super-brief comment that the functions called here are arbitrarily chosen, and just for sanity-testing?

Reply:
Done.